include/rnp/rnp2.h

```diff
@@ -237,6 +237,13 @@ rnp_result_t rnp_key_get_primary_uid(rnp_key_handle_t key, char **uid);
 rnp_result_t rnp_key_get_uid_count(rnp_key_handle_t key, size_t *count);
 rnp_result_t rnp_key_get_uid_at(rnp_key_handle_t key, size_t idx, char **uid);
 
+rnp_result_t rnp_key_add_uid(rnp_key_handle_t key,
```

Comment:
As that's public API some comments could be helpful

Reply:
Done